codegen/smithy-ruby-codegen/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Support services migrating from AWS Query to AWS JSON or RPC v2.
```

Comment:
This AWS specific bit is kind of weird to have in our generic changelog.

Reply:
Yeah. I can change this "from AWS Query to RPC v2" - the query trait (AWS) is used generically now so that bit must stay.